### PR TITLE
`webview_flutter` cleanup

### PR DESCRIPTION
**Pull request by domesticmouse**
## Pre-launch Checklist

- [x] I read the [Effective Dart: Style] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Effective Dart: Style]: https://dart.dev/guides/language/effective-dart/style
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: webview_flutter/codelab_rebuild.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ steps:
            
            android {
           -    compileSdkVersion flutter.compileSdkVersion
-          +    compileSdkVersion 32
```
Comment (parlough) [inline webview_flutter/codelab_rebuild.yaml:49]: Any context on why this needs a modify comment? Can the `compileSdkVersion` just stick to `flutter.compileSdkVersion`, with just the `minSdkVersion` needing to be changed?

File: webview_flutter/codelab_rebuild.yaml [hunk 2]
```diff
@@ -46,7 +46,7 @@ steps:
...
+          +    compileSdkVersion 32          // MODIFY
```
Comment (parlough) [inline webview_flutter/codelab_rebuild.yaml:49]: Any context on why this needs a modify comment? Can the `compileSdkVersion` just stick to `flutter.compileSdkVersion`, with just the `minSdkVersion` needing to be changed?

Comment (domesticmouse) [reply webview_flutter/codelab_rebuild.yaml:49]: No, it dies at runtime if memory serves. I don't know why it needs that SDK Version, but it does.

Comment (parlough) [reply webview_flutter/codelab_rebuild.yaml:49]: I see. Sounds good to me then :)

**Review by miquelbeltran — APPROVED** [review]
LGTM

**Pull request merged**